engine/execution/computation/manager_test.go

```diff
@@ -103,3 +104,40 @@ func TestComputeBlockWithStorage(t *testing.T) {
 	require.Len(t, returnedComputationResult.StateSnapshots, 1+1) // 1 coll + 1 system chunk
 	assert.NotEmpty(t, returnedComputationResult.StateSnapshots[0].Delta)
 }
+
+func TestExecuteScript(t *testing.T) {
+
+	t.Run("scripts do not store programs", func(t *testing.T) {
```

Comment:
I can't tell which line of this tests asserts that "scripts do not store programs"...

Reply:
Ah, that was a left-over from earlier code. The test ensured that programs that were stored during script execution would not be committed, i.e. used for other scripts or even transactions.

The PR now does not implement this anymore, so I removed the confusing subtest name. The test of executing a script on its own is still IMHO useful, so I kept it.